docs/index.html

```diff
@@ -51,6 +51,7 @@ <h2 class="title">A 1x Engineer...</h2>
           <li>Doesn't need to be passionate about the code they write or the problems they solve, but may be.</li>
           <li>Doesn't act surprised when someone doesn’t know something.</li>
           <li>Is willing and able to collaborate with others.</li>
+          <li>Willing to spend time help teammate get unstuck even when own tasks are time pressing.</li>
```

Comment:
```suggestion
          <li>Willing to spend time and energy to help teammates get unstuck, when appropriate.</li>
```

Reply:
Change is ok by me, keeps the same idea but rephrased.